src/backend/cgelem.c

```diff
@@ -3170,7 +3180,10 @@ elem * elstruct(elem *e, goal_t goal)
                     goto L1;
                 }
             }
-            if (I64 && (ty == TYstruct || (ty == TYarray && config.exe == EX_WIN64)))
+            if (I64 && (ty == TYstruct ||
+                       (ty == TYarray && config.exe == EX_WIN64) ||
+                       (0 && ty == TYarray && !tysimd(e->Ety)))
```

Comment:
remove that `0 &&` ? Git has full history anyway no need for dead code to stay around.